metron-platform/metron-solr/src/main/java/org/apache/metron/solr/writer/SolrWriter.java

```diff
@@ -33,17 +39,19 @@
 
 import java.io.IOException;
 import java.io.Serializable;
-import java.util.List;
-import java.util.Map;
+import java.lang.invoke.MethodHandles;
+import java.util.*;
```

Comment:
Style police checking in.  No \* please.

Reply:
You got it :)

Reply:
UGh, it slipped back in again. Ok, removing it.